.src/docker_build/main.py

```diff
@@ -20,41 +21,51 @@
 
 from settings import conf
 
-cli = typer.Typer()
+
+class Platform(str, Enum):
+    LINUX_AMD64 = "linux/amd64"
+    LINUX_ARM64 = "linux/arm64"
+
+
+build_cli = typer.Typer()
```

Comment:
what about removing `_cli` from these and prefixing the actual functions with `_`?

Reply:
Sounds like a good idea, will change that accordingly.

Reply:
Changed now. Also update pydantic, as I noticed it was slightly outdated.